CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file.
 
+### 0.0.3 (2022-05-15)
+
+## Bug Fixes
+
+* Creation of an empty conf file is not allowed ([#65](https://github.com/Certora/vscode-certora-prover/pull/65))
+* Choosing 'Choose file' as a file name is not an option ([#65](https://github.com/Certora/vscode-certora-prover/pull/65))
```

Comment:
The bug was that someone could choose the option `Choose file` as a solidity/spec file from the file picker.
so it was an `option` instead of a `placeholder`